tests/test_skills/test_apy_estimation_abci/test_apis.py

```diff
@@ -49,7 +50,7 @@
 SubgraphResponseType = Dict[str, ResponseItemType]
 
 
-def make_request(
+def __make_request_no_retries(
```

Comment:
for a function this does not matter, but please refrain from using double underscores generally, single underscore marks private and should be clear to other devs. Particularly can be annoying when something if used in methods without actual need for such, leads to name mangling during inheritance (you're basically saying, even child classes should not access this)